aries-site/src/pages/components/radiobuttongroup.js

```diff
@@ -21,13 +35,106 @@ const RadioButtonGroup = () => {
           <SubsectionText>{page.description}</SubsectionText>
           {page.status && <Status status={page.status} />}
           <Example
-            code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/input/RadioButtonGroupExample.js"
+            code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/radiobuttongroup/RadioButtonGroupExample.js"
+            designer="https://designer.grommet.io/radiobuttongroup?id=HPE-design-system-hpedesignsystem-hpe-com"
+            figma="https://www.figma.com/file/aZyY606ENQedz4FXugdzgS/HPE-Radio-Button-Group-Component?node-id=105%3A643"
             docs="https://v2.grommet.io/radiobuttongroup?theme=hpe#props"
+            width="medium"
           >
             <RadioButtonGroupExample />
           </Example>
         </Subsection>
       </ContentSection>
+      <ContentSection>
+        <Subsection name="Guidance">
+          <SubsectionText>Keep labels concise and simple.</SubsectionText>
```

Comment:
This feels a little awkward/orphaned to have separated from the other best practices.

Reply:
Good call. Will move into lead of best practices.

Reply:
implemented